ChocoPacker.UI/app/index.html

```diff
@@ -10,7 +10,62 @@
 <header class="app-header">
   <h1 class="logo">ChocoPacker</h1>
 </header>
```

Comment:
Data-ng-include should handle included template. It's bad idea to put anything inside div with this directive.